impl/src/main/java/org/glassfish/soteria/identitystores/LdapIdentityStore.java

```diff
@@ -412,6 +445,24 @@ private static String ensureFilterHasParens(String filter) {
         }
     }
 
+    private static void processNamingException(NamingException e) throws Exception {
```

Comment:
I know this was at least partly my idea -- sorry! -- but I don't think it's really needed. It's probably easier to just handle the exceptions directly.